src/ports/postgres/modules/deep_learning/input_data_preprocessor.sql_in

```diff
@@ -209,25 +214,39 @@ validation_preprocessor_dl(source_table,
     validation_preprocessor_dl() contain the following columns:
     <table class="output">
       <tr>
-        <th>buffer_id</th>
-        <td>INTEGER. Unique id for each row in the packed table.
+        <th>independent_var</th>
+        <td>BYTEA. Packed array of independent variables in PostgreSQL bytea format.
```

Comment:
We can also add that the independent var is first normalized and then packed and converted to bytea. I understand that we already mentioned this in the function definition. But I believe that it will be valuable to mention it again just like we do for dependent_var

Reply:
added this comment